<launch>
	<group ns="/seabee3">
		<include file="$(find seabee3_launch)/seabee3_driver_base.launch" />
		<include file="$(find seabee3_launch)/seabee3_driver_tuned.launch" />
	</group>

	<group ns="/xsens">
		<include file="$(find seabee3_launch)/xsens_node.launch" />
	</group>
</launch>
